dev/com.ibm.ws.microprofile.config.1.1_fat/fat/src/com/ibm/ws/microprofile/config/fat/tests/ClassLoadersTest.java

```diff
@@ -47,7 +47,7 @@ public class ClassLoadersTest extends FATServletClient {
     public static final String APP_NAME = "classLoaders";
 
     @ClassRule
-    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP50, MicroProfileActions.MP14, MicroProfileActions.MP41);
+    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP60, MicroProfileActions.MP14, MicroProfileActions.MP41);
```

Comment:
Should MP50 be added to the end of the method?

Reply:
I was not going to because the MP config implementation is the same for MP50 and MP60, so it doesn't seem necessary to run both, even in full mode, for functionality which is wholly within MP config, though I was going to run that past Tom once he's back.

The tests I was going to keep running on both were:
* the `BasicConfTests` test above
* the `mpConfig-3.0` TCK (once that's fixed to work against EE10)